packages/web-components/stencil.config.ts

```diff
@@ -52,6 +52,7 @@ export const config: Config = {
     },
     {
       type: 'www',
+      dir: process.env.ENV === 'production' ? '/storybook' : '',
```

Comment:
probably a silly question but where are we reading our env variables from? I looked around and couldn't determine where we may have `ENV` set. I didn't see it in any github action.

Reply:
@micahchiang Good point. While the PR is being reviewed, I've been trying to find an answer regarding adding environment variables.